### ISSUE TITLE
https://www.yahoo.com/ - Page blocking - iOS

### ISSUE THREAD

**Issue by veronikasav** [AdguardTeam/AdguardFilters#5355]
https://www.yahoo.com/
https://www.yahoo.com/news/otto-warmbier-isn-t-going-041157233.html

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/27292111/27239853-4cfdc2b8-52db-11e7-9a26-72af0e00275e.png)

Any article is blocked after the redirection
![image](https://user-images.githubusercontent.com/27292111/27239865-5b6dcadc-52db-11e7-98e6-15bce9e03fc8.png)

</details>

***System configuration***

Application version: 1.2.1.71

Device: iPhone
Platform: iPhone OS
OS: 9.3.2

Filters subscriptions:
ID=0 Name="User filter" Version=1.0.0.0 Enabled=YES
ID=2 Name="English filter" Version=1.1.21.87 Enabled=YES
ID=3 Name="Spyware filter" Version=1.0.12.77 Enabled=YES
ID=4 Name="Social media filter" Version=1.0.20.55 Enabled=YES
ID=14 Name="Annoyances filter" Version=1.0.14.19 Enabled=YES
ID=1 Name="Russian filter" Version=1.0.52.62 Enabled=YES
ID=5 Name="Experimental filter" Version=1.0.26.65 Enabled=YES
ID=11 Name="Mobile Ads filter" Version=1.0.8.74 Enabled=YES
ID=12 Name="Safari filter" Version=1.0.3.49 Enabled=YES

Ticket ID: 1313547

**Comment by veronikasav** [comment]
@Alex-302 English filter causes it. When it is turned off the pages load.

**Comment by Alex-302** [comment]
Strange. I can't reproduce it on any device.
@zebrum please check it on iPhone.

**Comment by dzamyshlyayeva** [comment]
a user turned filters one by one and here are the results: 
"The English filter is the one that causes it. When I turn it off the pages load and obviously all the ads are there as well. I tried all the other filters and turning them off didn't help"

**Comment by zebrum** [comment]
@Alex-302 can not reproduce now.

**Comment by Alex-302** [comment]
Ok then.
@veronikasav please ask user about this site.

**Comment by kbahlyk** [comment]
Yahoo broke again
****
![image](https://user-images.githubusercontent.com/29478021/27765587-24fc9160-5ebe-11e7-83d2-4015a9511017.png)

**Comment by Alex-302** [comment]
Still can't reproduce.
Try
```
@@||us.yahoo.com^$document
@@||www.yahoo.com^$document
```

**Comment by kbahlyk** [comment]
Forwarded to the customer.

**Comment by dzamyshlyayeva** [comment]
the user responded: "Unfortunately, all the sponsored ads returned when I added those."

**Comment by veronikasav** [comment]
I apologize for reopening this issue but some pages are still blocked. Could not reproduce this issue. 
Should I ask this customer for log files or anything?

**Comment by Alex-302** [comment]
Logs are useless here. 

Check:
*
```
@@||us.yahoo.com^$generichide
@@||www.yahoo.com^$generichide
```
* 
```
@@||us.yahoo.com^$urlblock
@@||www.yahoo.com^$urlblock
```

**Comment by veronikasav** [comment]
Checking.

**Comment by veronikasav** [comment]
He says: "Those rules just disable all ad blocking on yahoo"

Also, we have noticed that the shortcut menu doesn't work on this website. When you click on "Report Missed ad/Report Incorrect Blocking" it doesn't send any report.

**Comment by Alex-302** [comment]
> He says: "Those rules just disable all ad blocking on yahoo"

which rules? The first pair or the second?

**Comment by veronikasav** [comment]
I tried them both separately and pages still don't load. 

I did stumble upon something that might be beneficial in helping you to fix the issue. It seems the issue is with yahoo news sites...see screenshots below. If the story on the yahoo homepage takes you to a story that's on the yahoo finance or yahoo beauty banner for instance, it loads fine. However, if the story takes you to a story under the yahoo news banner, it doesn't load, even after refreshing.

<details><summary>Screenshot:</summary>

![screen shot 2017-07-27 at 2 13 19 pm](https://user-images.githubusercontent.com/27292111/28667826-d5ddc6cc-72d5-11e7-99ad-179d011ad3d5.png)

![screen shot 2017-07-27 at 2 13 36 pm](https://user-images.githubusercontent.com/27292111/28667830-d97a403a-72d5-11e7-8ec3-b97d8baff790.png)


</details>
